### PR TITLE
TestSendToDevice: add test to verify sending messages to a device works

### PR DESCRIPTION
**Pull request by sumnerevans**
Signed-off-by: Sumner Evans <sumner@beeper.com>

### CODE REVIEW

**Comment by richvdh** [comment]
hum. I'm afraid this seems to be a subset of #412. @S7evinK, could you take a look to see if there is anything we should use?

**Review by kegsay — APPROVED** [review]

File: internal/client/client.go [hunk 1]
```diff
@@ -195,6 +195,16 @@ func (c *CSAPI) SendEventSynced(t *testing.T, roomID string, e b.Event) string {
 	return eventID
 }
 
+// SendToDevice sends an event of type `eventType` to `userID`'s device `deviceID` with `content`.
+func (c *CSAPI) SendToDevice(t *testing.T, eventType, userID, deviceID string, content map[string]interface{}) *http.Response {
+	return c.MustDoFunc(t, "PUT",
+		[]string{"_matrix", "client", "v3", "sendToDevice", eventType, strconv.Itoa(c.txnID)},
```
Comment (kegsay) [inline internal/client/client.go:201]: Where is `c.txnID` incremented?

Comment (S7evinK) [reply internal/client/client.go:201]: FYI: I'm using the changes from `internal/client/client.go` in #412 and fixing this.

**Comment by kegsay** [comment]
Closing as stale.

**Pull request closed without merging**